src/themes/shared/zIndex.ts

```diff
@@ -0,0 +1,8 @@
+export const zIndices = {
+  hide: -1,
+  auto: 'auto',
+  base: 0,
+  menu: 1500,
+  overlay: 1300,
+  modal: 1400,
```

Comment:
Thank you for adding new shared tokens.

Regarding the file name I think, it should be `zIndices` to be in sync with the tokens export.
I agree with Colin about renaming the keys but also checking the values that we need as our tokens. I also don't see the reference to the figma tokens about the `zIndices`-> https://www.figma.com/file/WmNmJ7jdB0Z7tOjj516nFB/01.-AS24---Tokens?node-id=3%3A10

Would you mind doble checking with the designer and unify these tokens? Thank you 🙏